workshops/views.py

```diff
@@ -937,7 +937,6 @@ def all_events(request):
 
 
 @login_required
-@permission_required('workshops.add_todoitem', raise_exception=True)
```

Comment:
`if request.method == "POST" and request.user.has_perm('workshops.add_todoitem'):`
Is this fine (this returns the event details page) or should we return HTTP403 when the user doesn't have the permission?